src/Microsoft.AspNet.Razor/Parser/ParserContext.cs

```diff
@@ -208,6 +208,14 @@ public void OnError(SourceLocation location, string message)
             _errorSink.OnError(location, message);
```

Comment:
Make the above overload just call into your newly created one:

```C#
public void OnError(SourceLocation location, string message)
{
    // Default error length is 1 unless specified otherwise.
    OnError(location, message, length: 1);
}
```